File: clients/typescript/examples/collections-drop-collection.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db("**API_ENDPOINT**", {
  token: "**APPLICATION_TOKEN**",
});

// Drop a collection
(async function () {
  await database.dropCollection("**COLLECTION_NAME**");
})();

File: clients/csharp/examples/collections-drop-collection.cs
using DataStax.AstraDB.DataApi;
using DataStax.AstraDB.DataApi.Core;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Get a database
    var client = new DataAPIClient();
    var database = client.GetDatabase(
      "**API_ENDPOINT**",
      "**APPLICATION_TOKEN**"
    );

    // Drop a collection
    await database.DropCollectionAsync("**COLLECTION_NAME**");
  }
}
